src/main.rs

```diff
@@ -91,6 +94,7 @@ fn main() {
         kernel,
         cmdline,
         disks,
+	console,
```

Comment:
Before you publish a PR as well as doing any manual testing it's usually a great idea to do some preemptive "cleanup" steps. Our Travis CI enforces these but it's always good to be ahead of the curve:

`cargo clippy --all-targets --all-features -- -D warnings` will highlight any issues from the clippy linter (we currently require our code to be clippy clean.)

and  `find . -name "*.rs"  | xargs rustfmt` will nicely format the code.

Reply:
Ran these commands. Didn't get any warning. The code is now formatted.